server/manager/src/main/java/org/apache/accumulo/manager/split/Splitter.java

```diff
@@ -191,6 +186,29 @@ public FileInfo getCachedFileInfo(TableId tableId, TabletFile tabletFile) {
   }
 
   public void initiateSplit(SeedSplitTask seedSplitTask) {
-    splitExecutor.execute(seedSplitTask);
+    // Want to avoid queuing the same tablet multiple times, it would not cause bugs but would waste
+    // work. Use the metadata row to identify a tablet because the KeyExtent also includes the prev
+    // end row which may change when splits happen. The metaRow is conceptually tableId+endRow and
+    // that does not change for a split.
+    Text metaRow = seedSplitTask.getExtent().toMetaRow();
+    int qsize = queuedTablets.size();
+    if (qsize < 10_000 && queuedTablets.add(metaRow)) {
```

Comment:
This is not something that needs to be changed now as it was already hard coded, but I am wondering if there would ever be a use case requiring tuning the size of max number of queued tablets that would make creating a property for this worthwhile

Reply:
Not sure if will need to be configurable, would like to wait for a need before making it configurable.  Seems like if it was configurable it would need metrics or log messages that help when changing the config.